### DISCUSSION TITLE
Guide to safe scraping?

### DISCUSSION THREAD

**Discussion by kavinklain** [UltimaHoarder/UltimaScraper#1617] [Q&A]
What are some good practices when scraping? Using a VPN(does it even matter if they just look at your credit card info)? Avoid more influential models thats explicitly state they dont want their content copied? Avoid sharing the meta folder that comes with the scrapes?

I just started using this and wanted to know some way to stay anon when doing this. 

Has there been anyone at least getting banned? For what I'm aware onlyfans and models DO know this scraper exists. Sorry if this sounds like I'm getting cold feet, but I felt I wouldn't be the only one wondering this as more people will probably start coming here as we approach the Onlyfans Porn Ban in Oct

PS: i haven't tried, but if I scrape the same model again and they have new posts, will the scraper just scrape everything again and overwrite all the dupes?

**Comment by DIGITALCRIMINAL** [comment]
Just don't share the metadata folder and you'll be fine.

**Reply by kavinklain** [reply]
sorry a bit off topic, but if i scrape a model page again after they post new content, will the scraper just scrape everything again and overwrite the dupes?

**Reply by DIGITALCRIMINAL** [reply]
By default the script will ignore all previously downloaded files.

But if you set the script to overwrite files, it will check if the file is a duplicate file before overwriting.
